# Runing-Error-of-llm2vec
meet some unexpected error when running mntp of llm2vec
